presentation/matrix_view.py

```diff
@@ -51,7 +53,8 @@ def make_colors(mesh: MeshResults, latency_deteriorated_ms: int, latency_failed_
                         "backgroundColor": "red",
                     }
                 )
-            elif column.latency_microsec.value > latency_deteriorated_ms * 1000:
+            # TODO: use latency_thresholds specific to given from-to agent pairs. In separate Pull Request
```

Comment:
Ah, why not in this one? It is kind of unpleasant to have all of the configuration mechanics, but not the code using the config.

Reply:
OK, adding to current PR